arch/sim/src/sim/posix/sim_alsa.c

```diff
@@ -462,9 +462,16 @@ static int sim_audio_configure(struct audio_lowerhalf_s *dev,
         priv->sample_rate = caps->ac_controls.hw[0] |
                             (caps->ac_controls.b[3] << 16);
         priv->channels    = caps->ac_channels;
-        priv->bps         = caps->ac_controls.b[2];
-        priv->frame_size  = priv->bps / 8 * priv->channels;
 
+        /* offload mode, bps keep default value */
+
+        priv->bps = 16;
```

Comment:
why not 8, @jinxiuxu

Reply:
> why not 8, @jinxiuxu
default value is set to 16 in sim_audio_initialize, and we want to keep it here.